framework/src/main/java/org/fulib/fx/FulibFxApp.java

```diff
@@ -406,7 +406,8 @@ public void refresh() {
             throw new IllegalArgumentException(error(1011).formatted(controller.getClass().getName()));
         }
         ReflectionUtil.resetMouseHandler(stage());
-        display(parent);
+        getTitle(controller).ifPresent(title -> stage.setTitle(formatTitle(title)));
```

Comment:
I think i've seen this exact line about 4 times now, can we add a helper? e.g. app.applyTitle(controller, stage)